assets/templates/partials/inputs/input-radio.tmpl

```diff
@@ -9,5 +9,14 @@
   {{ end }}
 >  
 {{ if .LocaleKey }}
-  {{ template "partials/label" .}}
+  <label
+    class="ons-radio__label"
+    for="{{.Id}}"
+    id="{{.Id}}-label"
+    >
```

Comment:
Line 16 - Match indentation of `>` with opening `<label`

Reply:
this is done